.golangci.yml

```diff
@@ -53,10 +53,16 @@ linters-settings:
     # simplify code: gofmt with `-s` option, true by default
     simplify: true
 
+issues:
+  exclude:
+  - ifElseChain
```

Comment:
I pre-emptively added these as other HashiCorp repositories have adopted at least the `ifElseChain` exclusion, and there was a fair bit of code here that used `singleCaseSwitch`.